metadata-ingestion/src/datahub/ingestion/source/state/stateful_ingestion_base.py

```diff
@@ -81,7 +79,7 @@ def validate_config(cls, values: Dict[str, Any]) -> Dict[str, Any]:
         if values.get("enabled"):
             if values.get("state_provider") is None:
                 values["state_provider"] = DynamicTypedStateProviderConfig(
-                    type="datahub", config=None
+                    type="datahub"
```

Comment:
let's make it explicit here
```suggestion
                    type="datahub", config={}
```